From: Christian Hofstaedtler <zeha@debian.org>
Date: Tue, 10 Oct 2017 15:07:11 -0300
Subject: Mark Gemspec-reproducible change fixing #784225, too

I think the UTC date change will fix the Multi-Arch not-same file issue,
too.

Signed-off-by: Antonio Terceiro <terceiro@debian.org>
Signed-off-by: Christian Hofstaedtler <zeha@debian.org>
---
 lib/rubygems/specification.rb | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/lib/rubygems/specification.rb b/lib/rubygems/specification.rb
index 2560324..07fdd30 100644
--- a/lib/rubygems/specification.rb
+++ b/lib/rubygems/specification.rb
@@ -1787,7 +1787,9 @@ def date= date
                 raise(Gem::InvalidSpecificationException,
                       "invalid date format in specification: #{date.inspect}")
               end
-            when Time, DateLike then
+            when Time then
+              Time.utc(date.utc.year, date.utc.month, date.utc.day)
+            when DateLike then
               Time.utc(date.year, date.month, date.day)
             else
               TODAY
